### PR TITLE
v5.16 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Comment by amyblais** [comment]
@mkraft Can you help with this information for v5.16 changelog:
 - Any new config.json settings
 - Any API changes
 - Any Websocket Event changes
 - Any Database changes

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,148 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Improved perceived performance of the emoji picker.
+ - Improved post list performance by making thread comments be loaded only when needed.
+ - Improved quick switcher experience to make the autocomplete feel more like a modal rather than a dropdown.
+
+#### Administration
+ - Added the ability for system admins to revoke all sessions from all users.
+ - Added support for System Administrators to make public channels private and private channels public within the **System Console > User Management > Channel Configuration** page.
+ - Added user Id information in the **System Console > Users** page.
+ - Updated System Console plugin settings page to expose enable/disable setting.
+
```
Comment (wiersgallak) [inline source/administration/changelog.md]: - Added ability for system admins to view a user's team memberships and add to additional teams within **System Console > User Management > User Configuration**

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,148 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+### Highlights
+
+#### Guest Accounts
+ - Provides a controlled and secure method for users outside of an organization to collaborate with their organization without allowing the guest to access proprietary or confidential information.
+
+#### Plugin Marketplace
+ - The integrations marketplace is built into the product and gives system administrators the ability to discover and install Mattermost plugins that are compatible with the server version you are running.  
+
+#### Improved team/channel management
```
Comment (wiersgallak) [inline source/administration/changelog.md]: This was added in 5.14 https://docs.mattermost.com/administration/changelog.html#system-console-tools-to-manage-ldap-groups-within-teams-and-channels-ee.  
We could replace with:  
**Improved user management** - System Admins can view a user's team memberships and add a user to additional teams from the System Console without having to be a member of the team.

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,6 +4,157 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - System Administrators can view a user's team memberships and add a user to additional teams from the System Console without having to be a member of the team.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added support for showing TIF image thumbnail previews.
+ - Added the ability to remove the custom branding image.
+ - Added support for showing channel links as links in email notifications.
+ - Added support for direct message permalinks.
+ - Added a count for pinned posts header icon.
```
Comment (esethna) [inline source/administration/changelog.md]: This didn't make it. Server went in but not webapp

**Pull request merged**